install_config/topics/gce_scopes_permissions.adoc

```diff
@@ -0,0 +1,82 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+[id='configuring-gcp-permissions_{context}']
+= Configuring authorization for Google Cloud Platform
+
+*Roles*
+
+Configuring GCP for {product-title} requires the following GCP role:
+
+[cols='1,3']
+|===
+
+|`roles/owner`
+|Needed for creating service accounts, cloud storage, instances, images,
+templates, Cloud DNS entries, and to deploy load balancers and health checks.
+|===
+
+`delete` permissions might also be required if the user is expected to be
+redeploying the environment during testing phases.
+
+You can also create a service account to avoid using personal users when
+deploying GCP objects.
+
+See link:https://cloud.google.com/iam/docs/understanding-roles[the Understanding roles section of the GCP documentation] for more information, including steps for how to configure roles.
+
+*Scopes and service accounts*
+
+GCP uses scopes to determine if an authenticated identity is authorized to
+perform operations within a resource. For example, if application A with a
+read-only scope access token can only read, while application B with a
+read-write scope access token can read and modify data.
+
+The scopes are defined at the GCP API level as `https://www.googleapis.com/auth/compute.readonly`.
+
+Scopes can be specified using the `--scopes=[SCOPE,...]` option, or you can use
```

Comment:
Where do you specify these scopes?
s/Scopes can be specified using/You can specify scopes by using

Reply:
Why would you want to create an instance without scopes?

Reply:
You specify the scopes at instance creation time. And you can avoid using scopes if you don't want the instance to reach any GCP API.